### PR TITLE
chore: Fixed a typo in the name of the updatePullRequest method

### LINKED ISSUE #362: typo updatePullRequst vs updatePullRequest
seems to be a typo in the method name

http://michael.github.io/github/docs/2.3.0/Repository.html#updatePullRequst

### PR DESCRIPTION
**Pull request by AurelioDeRosa**
fixes #362

### CODE REVIEW

File: lib/Repository.js [hunk 1]
```diff
@@ -491,7 +491,7 @@ class Repository extends Requestable {
     * @param {Requestable.callback} [cb] - will receive the pull request information
     * @return {Promise} - the promise for the http request
     */
-   updatePullRequst(number, options, cb) {
+   updatePullRequest(number, options, cb) {
```
Comment (clayreimann) [inline lib/Repository.js]: I'm guessing that you'll need to update some tests because of this. (Also this is a breaking change)

Comment (AurelioDeRosa) [inline lib/Repository.js]: I don't think we have tests for this. This was added a few days back.

I agree this is BC but it seemed too much to need to do a bump of a major version for a typo.

Comment (mtscout6) [inline lib/Repository.js]: If you do an alias add a log that will be displayed regardless of the debug preferences that this is deprecated so you can remove it in subsequent major releases.

Comment (AurelioDeRosa) [inline lib/Repository.js]: This has been  open for a while now. I'm going to add an alias and add a warning as suggested.

**Pull request closed without merging**